packages/legend-transac/src/@types/event/events.ts

```diff
@@ -53,6 +61,7 @@ export interface EventPayload {
  * Represents the available events in the system.
  */
 export const microserviceEvent = {
+    'PAYMENTS.CANCEL_PRE_PURCHASE_RESERVATION': 'payments.cancel_pre_purchase_reservation',
```

Comment:
This is the section of the test-events, place this event below this section, after:
```text
/////////////////////////
```

Reply:
changed